### DISCUSSION TITLE
Bugs

### DISCUSSION THREAD

**Discussion by 1337kat** [ORelio/Minecraft-Console-Client#1937] [Ideas]
Please update the console client's /move command. I'd like to farm these bots by killing them and attracting them to my position or a specified spot. Furthermore, while using the /move command, the bot has a number of flaws, including the inability to do simple tasks such as hopping over a block or falling. When using /move xyz cordinates, the bot is limited to specific distances and cannot walk more than 12 blocks, which needs to be fixed. Last but not least, could you guys implement the /follow player command. - Thanks.

**Comment by Daenges** [comment]
These are good ideas. I already mentioned some problems, that make improvements difficult [here](https://github.com/MCCTeam/Minecraft-Console-Client/issues/1934#issuecomment-1034914332).
